args.go

```diff
@@ -44,6 +46,9 @@ func parseArgs() (common.Config, error) {
 			PlaceHolder("file.csv").
 			Short('o').
 			String(),
+		CPUs: app.Flag("cpus", "Maximum number of logical CPUs").
+			Default(strconv.Itoa(runtime.NumCPU())).
```

Comment:
I'm a bit ify about setting the default value to `runtime.NumCPU` as it is possible that someone running Gocannon might set the desired value via `GOMAXPROCS` env variable. Consequently, such setting would be overwritten. Instead, I would suggest not setting the Default value at all, so that it is initialized with 0, which will not alter the existing value when passed to `runtime.GOMAXPROCS()`. Ditching the default flag value would require modifying `printHeader()` so that it retrieves the value from `GOMAXPROCS()` if the config value equals 0.